examples/workspace/.bazelrc

```diff
@@ -0,0 +1,46 @@
+# A bzlmod example will be provided separately
+common --noenable_bzlmod
+
+
+# Use a minimal set of environment variables for action execution to improve
+# build hermeticity and remote cache hits.
+common --incompatible_strict_action_env
+
+
+# Enable BuildBuddy build event service
+common --bes_results_url=https://app.buildbuddy.io/invocation/
+common --bes_backend=grpcs://remote.buildbuddy.io
+# Enable BuildBuddy Remote Execution
+common:remote --remote_timeout=3600
+common:remote --remote_executor=grpcs://remote.buildbuddy.io
+
+
+# Target Linux platform when build remotely
+#
+# Usage:
+#
+#   $ bazel build --config=linux //...
+#
+common:linux --config=remote
+common:linux --platforms=@buildbuddy_toolchain//:platform_linux
+# Register "execution platforms" and "cc toolchains" dynamically using Bazel flag.
```

Comment:
It's not so clear what "dynamically" means in this context.

Reply:
Gona removes "dynamically". I was trying to say that we gonna use automatic platform - toolchain resolution with this (instead of the old `--cpu` and `--crosstool_top` approach)